### PR TITLE
Updating doc to contain exact commands to be run for testing

### PR DESCRIPTION
**Pull request by aliceh**
In this commit, I updated the commands used in testing because of the recant changes i.e. logging in with backplane and having to elevate to cluster-admin for creating and describing resources.

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ To generate a self signed cert and key follow these [steps](https://www.linode.c
 
 Example of creating a secret and customdomain:
 ```
-oc new-project my-project
-oc create secret tls acme-tls --cert=fullchain.pem --key=privkey.pem
-oc apply -f <(echo "
+oc create ns my-project
+oc create secret tls acme-tls --cert=fullchain.pem --key=privkey.pem -n my-project  --as backplane-cluster-admin
+oc apply --as backplane-cluster-admin -f <(echo "
```
Comment (fahlmant) [inline TESTING.md:54]: `--as backplane-cluster-admin` is specific to SREP and shouldn't be included in public documentation

File: TESTING.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ To generate a self signed cert and key follow these [steps](https://www.linode.c
 
 Example of creating a secret and customdomain:
 ```
-oc new-project my-project
-oc create secret tls acme-tls --cert=fullchain.pem --key=privkey.pem
-oc apply -f <(echo "
+oc create ns my-project
```
[the remark this answers is not in the document]
Comment (aliceh) [reply TESTING.md:52]: this command does not work with backplane, it  logs you out and you have to log back  in

File: TESTING.md (hunk 1 above)
Comment (aliceh) [reply TESTING.md:54]: It is a valid point. @dustman9000, feel free to revert the changes. I'd be happy to add as an SOP or wiki doc the SRE-specific version of testing CDO.

Comment (dustman9000) [reply TESTING.md:54]: Valid point, but backplane specific documentation is already in place.

**Pull request merged**